README.md

```diff
@@ -62,6 +62,43 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
+## Unit Test
+
+The unit test scripts are under `./__test__` folder and tested functions are under `./src/store`.
+
+The tested functions are:
```

Comment:
no need to list every test, just add a summary of whats being tested.

Reply:
Yes, I just modified this part with a simple summary.